lib/Braintree/Http.php

```diff
@@ -166,9 +167,16 @@ public function _doUrlRequest($httpVerb, $url, $requestBody = null)
         curl_close($curl);
         if ($this->_config->sslOn()) {
             if ($httpStatus == 0) {
-                throw new Exception\SSLCertificate();
+                if ($error_code == 35) {
+                    throw new Exception\SSLCertificate($error, $error_code);
+                } else {
+                    throw new Exception\Connection($error, $error_code);
```

Comment:
This looks like a breaking change, given that we document the existing behavior [here](https://developers.braintreepayments.com/reference/general/exceptions/php#ssl-certificate). However, it's a great addition to add additional information to the exception. Would you mind changing this back to an `Exception\SSLCertificate`?